templates/projects/task_presenter_editor.html

```diff
@@ -79,20 +79,26 @@
     <ul class="nav nav-tabs editors col-md-12" role="tablist" id="tabs">
         <li id="tab-nav-guidelines" role="presentation"><a href="#tab-content-guidelines" aria-controls="basics" role="tab" data-toggle="tab">Instructions</a></li>
         <li id="tab-nav-presenter" role="presentation"><a href="#tab-content-presenter" aria-controls="basics" role="tab" data-toggle="tab">Task Presenter</a></li>
-        <a id="help-btn" class="btn btn-info navbar-right " target="_blank" href="http://docs.pybossa.com"><i class="fa fa-book"></i> {{ _('Help') }}</a>
+        <a id="help-btn" class="btn btn-info navbar-right" target="_blank" href="http://docs.pybossa.com"><i class="fa fa-book"></i> {{ _('Help') }}</a>
+        <li id="expand-btn" class="tab-nav-collapse-expand navbar-right tab-link" onclick="expandTaskPresenter()" id ="component-builder-description">
+          <a class="cursor-link" title="F11 or ESC to collapse">
```

Comment:
Is there a shortcut for Mac user?

Reply:
Fn + F11 doesn't work to me. I did some research and I think it is because of Mac system conflict. So the key shortcut does't work for Mac and they really need the button you developed.